src/components/RelativeRangeDatePicker/RelativeRangeDatePicker.tsx

```diff
@@ -42,6 +42,8 @@ export interface RelativeRangeDatePickerProps
     format?: string;
     /** A placeholder date that controls the default values of each segment when the user first interacts with them. Defaults to today's date at midnight. */
     placeholderValue?: DateTime;
+    /** Show dates hint on calendar button hover **/
+    withDatesHint?: boolean;
```

Comment:
I think, the prop should hide the hint for better backward compatibility and it name should be "hideDatesHint".

Reply:
The thing is the component accepts all the boolean properties with `show` prefix
Also, by default the hint will be shown, this guarantees backward compatibility, because the condition is ` withDatesHint !== false`, not `withDatesHint`

Reply:
But yes, hideDatesHint looks more logical.